bioio_imageio/metadata_utils.py

```diff
@@ -0,0 +1,50 @@
+from typing import Union
+
+
+def generate_ome_image_id(image_id: Union[str, int]) -> str:
```

Comment:
how many repos has this been copied and pasted to? sort of tempted to say this should live in `bioio_base`

Reply:
I've been wondering this too, might be a good move for the next `bioio_base` version